internal/loader/configuration_setting_loader.go

```diff
@@ -664,7 +664,15 @@ func deduplicateFilters(filters []acpv1.Selector) []acpv1.Selector {
 				}
 			}
 			if !findDuplicate {
-				result = append(result, filters[i])
+				labelFilter := filters[i].LabelFilter
```

Comment:
I don't think we should do the conversion while deduplicating the selectors, we need to do it more straightforwardly.

Reply:
And what about the label for the `sentinel`?